<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>471</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>文件选择</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="SelectButton">
          <property name="text">
           <string>select</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>编辑区</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Save to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ExecuteButton">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ExitButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>9</y>
      <width>501</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>数据展示区</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labela">
          <property name="text">
           <string>a:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_a"/>
        </item>
        <item>
         <widget class="QLabel" name="labelb">
          <property name="text">
           <string>b:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_b"/>
        </item>
        <item>
         <widget class="QLabel" name="labelc">
          <property name="text">
           <string>c:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_c"/>
        </item>
        <item>
         <widget class="QLabel" name="labeld">
          <property name="text">
           <string>d:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_d"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>图片展示区</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>chl1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl11</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>419</width>
                <height>159</height>
               </rect>
              </property>
              <widget class="QLabel" name="imagelabel1">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>391</width>
                 <height>161</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>chl2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>chl11</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>419</width>
                <height>159</height>
               </rect>
              </property>
              <widget class="QLabel" name="imagelabel2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>391</width>
                 <height>161</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_O">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="newAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>另存为(&amp;F)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
